docs/manual/src/Upgrading.md

```diff
@@ -0,0 +1,37 @@
+# v0.28.x -> v0.29.x
+
+## Custom types
+
+Custom types are now implemented using a macro, rather than implementing the
+`UniffiCustomTypeConverter` trait.
+
+Before:
+
+```rust
+impl UniffiCustomTypeConverter for MyType {
+    type Builtin = UniFFIBuiltinType;
+
+    fn into_custom(val: Self::Builtin) -> uniffi::Result<Self> {
+        ...
+    }
+
+    fn from_custom(obj: Self) -> Self::Builtin {
+        ...
+    }
+}
+```
+
+After:
+
+```
+uniffi::custom_type!(MyType, UniFFIBuiltinType, {
```

Comment:
If we are moving to the term "existing" then this example should maybe be change - eg, line 12 could say, eg, `type Builtin = AnExistingUniFFIType;`  and this line could use the same name - it avoids using the name "builtin" when describing the new world.